web/src/app/components/Modal/EditModal.tsx

```diff
@@ -0,0 +1,32 @@
+import { Modal, Button, Box } from '@mantine/core';
+import { IconXboxX } from '@tabler/icons-react';
+import { ReactNode } from 'react';
+import styles from '../../styles/Modal.module.css';
+
+type ModalProp = {
+  opened: boolean;
+  close: () => void;
+  content: ReactNode;
+};
+
+export default function EditModal({ opened, close, content,title }: ModalProp) {
+  return (
+    <Modal
+      opened={opened}
+      onClose={close}
+      closeButtonProps={{
+        icon: <IconXboxX size={20} stroke={1.5} color="#ffffff" />,
```

Comment:
Please make the icon size 50 and stoke 2 for the close button. It appears tiny in compared to the other icons.